libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/JSTestCase.java

```diff
@@ -79,23 +80,15 @@ public static void runJSTestSuite(String jsSuite, Iterable<String> testNames, in
             // Block until the javascript has notified the container that it's ready
             TestRunnerPlugin.readyForTests.take();
 
-            // Now run all the tests and collect the resuts in testResults
+            // Now run all the tests and collect the results in testResults
             for (String testName : testNames) {
-                final String jsCmd = "javascript:" + "navigator.testrunner.setTestSuite('" + jsSuite + "');" +
-                    "navigator.testrunner.startTest('" + testName + "');";
-                final CordovaWebView appView = activity.getAppView();
-                if (appView != null) {
-                    appView.getView().post(new Runnable() {
-                        @Override
-                        public void run() {
-                                appView.loadUrl(jsCmd);
-                        }
-                    });
-                }
+                final String jsCmd = "javascript:" + "navigator.testrunner.setTestSuite('" + jsSuite + "');" + "navigator.testrunner.startTest('" + testName + "');";
+                final CordovaWebView appView = activity.getCordovaWebView();
```

Comment:
This looks like another regression from the Gradle 8 and related updates on `dev`, since this test wouldn't compile somehow.  This source had a lot of inspector warnings also, alas!